readme.txt

```diff
@@ -195,6 +195,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= TBD
+
+* Tweak - Declared dynamic properties in Tribe__Tickets__Main, Tribe__Tickets__Tickets_Handler, Tribe__Tickets__REST__V1__Messages to prevent warnings in php 8.2 [ET-1950]
```

Comment:
This is a bit too specific:
```suggestion
* Tweak - Resolved multiple warning logs when using PHP v8.2 or higher. [ET-1950]
```

Reply:
In my 1:1 with @bordoni he wanted me to mention the words "dynamic", name the 3 classes and "PHP 8.2". I'm ok with any message here

Reply:
Okay. This is fine, then.